CONTRIBUTING.md

```diff
@@ -1,35 +1,211 @@
-# Filing an issue
+# How to contribute to qBittorrent
 
-### Must read
-* If you aren't sure, you can ask on the [**forum**](http://forum.qbittorrent.org) or read our [**wiki**](http://wiki.qbittorrent.org) first.
-* Do a quick **search**. Others might already reported the issue.
-* Write in **English**!
-* Provide **version** information: (You can find version numbers at menu `Help -> About -> Libraries`)
-  ```
-  qBittorrent:
-  Qt:
-  libtorrent:
-  boost:
-  OS version:
-  ```
-* Provide **steps** to reproduce the problem, it will be easier to pinpoint the fault.
-* **Screenshots**! A screenshot is worth a thousand words. just upload it. [(How?)](https://help.github.com/articles/file-attachments-on-issues-and-pull-requests)
+There are three main ways to contribute to the project.
+Read the respective section to find out more.
 
-### Good to know
-* **Be patient**. The dev team is small and resource limited. Devs finding their free time, analyzing the problem and fixing the issue, it all takes time. :clock3:
-* If you can code, why not become a **contributor** by fixing the issue and open a pull request? :wink:
-* Harsh words or threats won't help your situation. What's worse, your complain will (very likely) to be **ignored**. :fearful:
+#### Table Of Contents
+
+[Submitting an issue/bug report](#submitting-an-issuebug-report)
```

Comment:
Please add bullet point (and maybe boldface) for section titles

Reply:
Ok